lib/rubocop/cop/lint/unneeded_splat_expansion.rb

```diff
@@ -124,7 +124,9 @@ def remove_brackets(array)
           elements = elements.map(&:source)
 
           if array_start.start_with?(PERCENT_W)
+            # rubocop:disable Lint/NestedDoubleQuotesInInterpolation
             "'#{elements.join("', '")}'"
```

Comment:
As this is technically inside a single-quoted string it seems like a bug in the cop to me.